app/controllers/albums.js

```diff
@@ -11,5 +13,26 @@ module.exports = {
         })
       });
     });
+  },
+
+  buy(req, res, next) {
+    const purchase = {
+      userId: req.user.id,
+      albumId: req.params.id
+    };
+    Purchase.findOne({ where: purchase })
```

Comment:
We aren't checking if the album actually exists in the album provider.

Reply:
fixed